curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/643c8d9a73965c0a26d7f634.md

```diff
@@ -0,0 +1,250 @@
+---
+id: 643c8d9a73965c0a26d7f634
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
+
+The `topic_list` object contains several properties, but you only need the `topics` array.
+
+Destructure the `topics` array from the `topic_list` object.
+
+# --hints--
+
+You should use `const` to destructure the `topics` variable from the `topic_list` object.
+
+```js
+assert(code.match(/{\s+const\s+{topic_list, users}\s+=\s+data;\s+const\s+{\s*topics\s*}\s+=\s+topic_list/g));
```

Comment:
I am unable to pass this test with the correct code

<img width="865" alt="Screenshot 2023-11-16 at 12 41 43 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/681c8708-30c6-42ac-93ea-f586af4e4573">

Reply:
done